packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -917,8 +919,30 @@ export async function getAvailableSlots({ input, ctx }: GetScheduleOptions): Pro
     }
   }
 
+  const troubleshooterData = enableTroubleshooter
+    ? {
+        troubleshooter: {
+          // One that Salesforce asked for
+          askedContactOwner: contactOwnerEmailFromInput,
+          // One that we used as per Routing
+          usedContactOwner: contactOwnerEmail,
+          routedHosts: routedHostsWithContactOwnerAndFixedHosts.map((host) => {
+            return {
+              email: host.email,
+              user: host.user.id,
+            };
+          }),
+          hosts: eventHosts.map((host) => ({
+            email: host.email,
+            user: host.user.id,
+          })),
+        },
+      }
+    : null;
+
   return {
     slots: withinBoundsSlotsMappedToDate,
+    ...troubleshooterData,
```

Comment:
when `enableTroubleshooter` is false then we can't spread `troubleshooterData`

Reply:
`null` is spreadable.